src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/validator/GenerateUrlRequestValidatorImpl.java

```diff
@@ -124,7 +124,7 @@ private boolean isNewSubmission(InitialPackage initialPackage) {
      */
     private List<String> validateParties(InitialPackage initialPackage) {
 
-        if (StringUtils.isBlank(initialPackage.getCourt().getFileNumber()) && initialPackage.getParties().isEmpty()) {
+        if (StringUtils.isBlank(initialPackage.getCourt().getFileNumber()) && (initialPackage.getParties() == null || initialPackage.getParties().isEmpty())) {
```

Comment:
or ... `&& (CollectionUtils.isEmpty(initialpackage.getParties())`

Reply:
Yes, this method is being updated in FLA-895. Which I'm working on now